synapseAnnotations/data/experimentalData.json

```diff
@@ -12,6 +12,26 @@
     "columnType": "STRING",
     "maximumSize": 250,
     "enumValues": [
+      {
+        "value": "elevated plus maze test",
+        "description": "A method which utilizes an experimental apparatus consisting of four arms, usually two enclosed and two open, in the shape of a plus (+) upon which the test animal can walk. The maze is elevated a set distance, for example 40 to 60 centimeters, above the floor or platform upon which the apparatus sits."
+        "source": "http://purl.obolibrary.org/obo/MMO_0000262"
+      },
+      {
+        "value": "open field test",
+        "desription": "A test utilizing an enclosure consisting of a single area bordered by clear or opaque walls, used to measure movement, exploratory behavior, etc of an experimental subject."
+        "source": "http://purl.obolibrary.org/obo/MMO_0000258"
+      },
+      {
+        "value": "rotarod performance test"
+        "description": "A method that utilizes a device consisting of a rotating rod the speed of which is mechanically driven and precisely controlled, for instance held constant or accelerated. The rotarod test is often used to measure riding time or endurance in order to evaluate balance, motor coordination or grip strength."
+        "source": "http://purl.obolibrary.org/obo/MMO_0000567"
+      },
+      {
+        "value": "contextual fear conditioning test"
```

Comment:
Just to double-check, this is different than our existing "contextual conditioning behavior"?
https://github.com/Sage-Bionetworks/synapseAnnotations/blob/32dc52fa25970ac82a3aa46fb25c2d20eb944170/synapseAnnotations/data/experimentalData.json#L250-L252

Reply:
@amapeters could you follow up on this? We'd like to get all outstanding issues merged so we can do a new release of the annotations before the data release.

Reply:
@amapeters do we need this for the November data release?